python-frontend/src/main/java/org/sonar/plugins/python/api/PythonCheck.java

```diff
@@ -87,6 +89,14 @@ public List<IssueLocation> secondaryLocations() {
       return secondaryLocations;
     }
 
+    public void addQuickFix(PythonQuickFix quickFix){
```

Comment:
I would also mark those as "Beta": I think those API make only sense in SonarLint context where custom rules are not supported. So practically we don't expect anyone external to the company to use them.
Can you also add a comment that this API makes only sense for SonarLint and and it's not advisable to use it from a custom rule?